flare_portal/experiments/forms.py

```diff
@@ -246,6 +273,61 @@ def save(self) -> None:
         self.participants.delete()
 
 
+class VolumeIncrementsWidget(forms.MultiWidget):
```

Comment:
Just saw this in the docs. [SplitArrayField](https://docs.djangoproject.com/en/3.2/ref/contrib/postgres/forms/#splitarrayfield). Not sure, but this could be what we need.

Reply:
I've tried using a SplitArrayField, sadly it doesn’t support float fields, I also tried using the SplitArrayWidget which also has issues with parsing float fields. 
A MultiWidget seems like the best option here. I’ve tried to clear up some of your questions by reformating the code and adding comments too.

Reply:
Ah that's too bad. Thanks for looking into it!